modules/imap/functions.php

```diff
@@ -1412,6 +1412,56 @@ function snooze_dropdown($output, $unsnooze = false) {
     return $txt;
 }}
 
+/**
+ * @subpackage imap/functions
+ */
+if (!hm_exists('get_forward_content')) {
+    function get_forward_content($format, $only_label = false) {
+        if ($format == 'Forward as message attachment') {
+            $label = 'Forward as message attachment';
+        } elseif ($format == 'Edit as new message') {
+            $label = 'Edit as new message';
+        }
+        if ($only_label) {
+            return [$label];
+        }
+    }}
+
+/**
+ * @subpackage imap/functions
+ */
+if (!hm_exists('forward_content')) {
+    function forward_content() {
+        $values = array(
+            'Forward as message attachment',
+            'Edit as new message'
+        );
+        return $values;
+    }}
+
+/**
+ * @subpackage imap/functions
+ */
+if (!hm_exists('forward_dropdown')) {
+    function forward_dropdown($output,$reply_args) {
+        $values = forward_content();
+    
+        $txt = '<div class="dropdown d-inline-block">
+                    <button type="button" class="btn btn-outline-success btn-sm dropdown-toggle" id="dropdownMenuForward" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="true">'.$output->trans('Forward').'</button>
+                    <ul class="dropdown-menu" aria-labelledby="dropdownMenuForward">';
+        foreach ($values as $format) {
```

Comment:
no need to look just hardcode them as they are 2 values

Reply:
Please Review